member_database/static/js/latex_widget.js

```diff
@@ -1,56 +1,46 @@
-var seperateTex = function(string){
-  var stringArray = [];
-  var index = 0;
-  while(index > -1){
-    index = string.indexOf('$') > -1 ? string.indexOf('$') : string.length;
-    string.substring(0, index) ?
-      stringArray.push(string.substring(0, index)) : undefined;
-    string = string.substring(index + 1, string.length);
-    index = string.indexOf('$');
-    if(index > -1){
-      string.substring(0, index) ?
-        stringArray.push('$' + string.substring(0, index)) : undefined;
-      string = string.substring(index+1, string.length);
-    }
-  }
-  return stringArray;
+function count(s, item) {
+  return s.length - s.replaceAll(item, "").length;
```

Comment:
Ok, can also use split (more readable?). Insane that js strings don't have count.

Reply:
Yeah, I looked at several solutions, chose this one, none are great imho